src/core/utils/entropy_utils.c

```diff
@@ -29,15 +29,15 @@
 #include "entropy_utils.h"
 
 #include <mbedtls/entropy.h>
-#include <openthread/platform/random.h>
+#include <openthread/platform/entropy.h>
 
 int otrMbedtlsEntropyPoll(void *aData, unsigned char *aOutput, size_t aInLen, size_t *aOutLen)
 {
     (void)aData;
 
     int rval = 0;
     // TODO access to random SHOULD be guarded.
```

Comment:
Do we still need this TODO if we rely on mbedtls threading support?

Reply:
@gjc13, thoughts?

Reply:
TODO removed, see:
https://tls.mbed.org/kb/development/thread-safety-and-multi-threading